### ISSUE TITLE
Can't request Season 2 of TV show when Season 1 is available.

### ISSUE THREAD

**Issue by MorningSleeper** [Ombi-app/Ombi#3969]
**Describe the bug**

When I have season 1 of TV show named Dickinson in Plex, Ombi shows Available instead of partially available so there is no Request Button to request season 2. If I delete season 1 from Plex and force Ombi to update, the Request button appears and I can request both season 1 and/or 2. 

**To Reproduce**

First add season 1 of Dickinson and then try to add season 2

**Expected behavior**

Should show partially available not available since I only have season 1 and there should be a request button to request season 2.

**Screenshots**

![image](https://user-images.githubusercontent.com/8920632/104084491-4e2b4880-51fc-11eb-951a-20ccc39f60b8.png)

**Logs (Logs directory where Ombi is located)**
If applicable, a snippet of the logs that seems relevant to the bug if present.

**Desktop (please complete the following information):**

Windows 10 19042.685
Chrome Version 87.0.4280.141 (Official Build) (64-bit)
Firefox 84.0.2 (64-bit)

**Ombi Version (please complete the following information):**
Ombi 4.0.912.0 
Plex 1.21.1.3876
Database Type: SQLite

**Additional context**

**Comment by tidusjar** [comment]
I'm able to request it fine 
![image](https://user-images.githubusercontent.com/6642220/104445868-94ccca00-5591-11eb-81ed-abc0c991dbb1.png)

Please check again, it could be that TV Maze didn't have the information at the time you tried

**Comment by MorningSleeper** [comment]
Thanks for reply, I have narrowed down the conditions under which I am able to replicate the issue.

If I have all but the last aired episode, in this case season 2 episode 3, or anything less, the Request button appears and I can request any of the remaining episodes in Season 2.

![image](https://user-images.githubusercontent.com/8920632/104492527-a362b380-5588-11eb-867c-1f29670dfdc2.png)

If I have all of season 1 and all of season 2 that has aired including the most recently aired episode 3, there is no Request button to request episodes that have not aired yet like season 2 episode 4 airing on 1/15/20 and all others after that. It shows as Available instead of Partially Available.

![image](https://user-images.githubusercontent.com/8920632/104492395-6ac2da00-5588-11eb-80a6-1b788b0da0e6.png)

Strangely enough if I click the gear icon to far right of Report Issue button then I can submit a request on behalf of someone for any unaired episode.

**Comment by PaarthShah** [comment]
I have a similar problem: Can't request the third season since the first two are already available. Clicking the blue "Partially available" on this page does nothing. 
![image](https://user-images.githubusercontent.com/20485810/113071471-a463b400-9179-11eb-9223-1e6b65a8d53f.png)

**Comment by tidusjar** [comment]
You on the latest version? Since we are no longer using TvMaze as a data provider now

**Comment by tidusjar** [comment]
When selecting those two episodes in the grid, you should get a (+) button appear to request the remaining?

**Comment by PaarthShah** [comment]
> 
> 
> When selecting those two episodes in the grid, you should get a (+) button appear to request the remaining?

See my screenshot above: no such button appears. Tested on both Firefox and Chrome, Firefox android, and Chrome iOS.

**Comment by tidusjar** [comment]
I'll investigate

**Comment by MorningSleeper** [comment]
Might be related to the episodes we are trying to request have not aired yet as that is one similarity between our two similar but different reports.

In my case, if I had all episodes that had previously aired, I could not request any future ones even though they were listed. If I did not have the last aired episode, I could request the future ones listed.

I have since updated to latest release so my install is no longer using TV Maze so will need to retest again to see if I still have the same problem.
